src/frontend/ocamlmerlin/ocamlmerlin.c

```diff
@@ -9,6 +9,7 @@
 #include <windows.h>
 #include <Lmcons.h>
 #include <process.h>
+#include <Sddl.h>       // ConvertSidToStringSid
```

Comment:
```suggestion
#include <sddl.h>       // ConvertSidToStringSid
```

It's very niche, but if ocamlmerlin is cross-compiled from Linux, the mingw-w64 version of this header is `sddl.h` not `Sddl.h`!